### PR TITLE
Fix #85

### LINKED ISSUE #85: Linting should check pipeline names for guidelines
https://github.com/nf-core/ICGC-featureCounts/issues/8

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
#85 

Linting now checks whether the pipeline name only contains lower case & alphabetical characters. Otherwise a warning is displayed.
For that the minimal_working_example was renamed to minimalworkingexample

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [x] If you've fixed a bug or added code that should be tested, add tests!
 - [x] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/tools/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: nf_core/lint.py [hunk 1]
```diff
@@ -802,6 +803,21 @@ def check_pipeline_todos(self):
                                 l = '{}..'.format(l[:50-len(fname)])
                             self.warned.append((10, "TODO string found in '{}': {}".format(fname,l)))
 
+    def check_pipeline_name(self):
+        """Check whether pipeline name adheres to lower case/no hyphen naming convention"""
+        
+        #Extract last part of given path, a.k.a. the workflow name
+        workflow_name = os.path.basename(os.path.normpath(self.path))
```
Comment (ewels) [inline nf_core/lint.py]: This won't always work - for example, I call my forks of pipelines different things. But the good news is that we already have the pipeline name in the class for you to use 🎉 `self.pipeline_name`

Comment (FriederikeHanssen) [reply nf_core/lint.py]: Thanks for the review 😃 . Added the changes

**Comment by apeltzer** [comment]
```
E       AttributeError: 'NoneType' object has no attribute 'islower'
```
@FriederikeHanssen :-)

**Comment by FriederikeHanssen** [comment]
@apeltzer yeah , saw that. I will fix it  at the airport or tomorrow morning  :)

**Pull request merged**